app/helpers/reports/regions_helper.rb

```diff
@@ -36,4 +36,9 @@ def percentage_or_na(value, options)
   def cohort_report_type(period)
     "#{period.type.to_s.humanize}ly report"
   end
+
+  def recent_bp_log(blood_pressures)
+    blood_pressures
+      .order(Arel.sql("DATE(recorded_at at time zone 'utc' at time zone '#{CountryConfig.current[:time_zone]}') DESC, recorded_at ASC"))
```

Comment:
Why do we need to cast timezones to sort? Also, why are we doing a date desc sort, and then again an asc sort?

Reply:
Understood the desc date, asc time from the spec.

Reply:
@ssrihari it was very validating to know that someone went through the exact sequence of thoughts looking at this code 😂